apps/src/templates/teacherDashboard/AssignmentVersionMenuHeader.jsx

```diff
@@ -16,7 +16,9 @@ const style = {
     fontSize: 16,
     fontWeight: 'bold',
     height: rowHeight,
-    color: 'white'
+    color: 'white',
```

Comment:
nit: should we use `color.white` here? looks like it's already imported in this file.